pkg/backends/jmeter/resources.go

```diff
@@ -175,7 +175,12 @@ func (c *JMeter) NewTestdataConfigMap() ([]*coreV1.ConfigMap, error) {
 func (c *JMeter) NewPod(i int, configMap *coreV1.ConfigMap, podAnnotations map[string]string) *coreV1.Pod {
 	loadtest := c.loadTest
 	optionalVolume := true
-	WorkerConfig := loadtest.Spec.WorkerConfig
+
+	imageRef := fmt.Sprintf("%s:%s", loadtest.Spec.WorkerConfig.Image, loadtest.Spec.WorkerConfig.Tag)
+	if imageRef == "" {
```

Comment:
imageRef will never be empty

```suggestion
	imageRef := fmt.Sprintf("%s:%s", loadtest.Spec.WorkerConfig.Image, loadtest.Spec.WorkerConfig.Tag)
	if imageRef == ":" {
```

Reply:
Fixed via aec7525, thanks!